database/db.py

```diff
@@ -157,15 +162,15 @@ def createUser(self, parsedData):
         if(id == None):
             id = 1
         else:
-            id = str(id + 1)
+            id = str(eval(id) + 1)
```

Comment:
Is there a reason the eval is here?

Reply:
> Is there a reason the eval is here?

I Wass addressing errors that my IDE had detected and it was complaining about concatenating with an integer (meaning that id was being treated like a string). I assumed my IDE knew what it was talking about and converted that str into a number.

Reply:
Gotcha, wasn't really sure why the eval was there. the db fetch will always return an int since its pulling from  the db. Doesn't hurt to keep it.